# def be_cheerful(name='Vicky',repeat=2):
#   print(f"GoodMorning {name}\n"* repeat)
  
# be_cheerful("Felix")
# be_cheerful()
# be_cheerful("Benny Bob", 10)

#1
# def number_of_food_groups():
#     return 5
# print(number_of_food_groups())
# this returned just the number 5 because that is what is being called in the function


#3

# def number_of_books_on_hold():
#     return 5
#     return 10
# print(number_of_books_on_hold())
#it is returning 5 only due to it being the first before "return 10", the code will never see that once a return call is there

#4
# def number_of_fingers():
#     return 5
#     print(10)
# print(number_of_fingers())
#i thought it would still print out 10, i was wrong it would only still return 5

# #6
# def add(b,c):
#     print(b+c)
# print(add(1,2) + add(2,3))


# #7
# def concatenate(b,c):
#     return str(b)+str(c)
#     print(concatenate(2,5))


#h

